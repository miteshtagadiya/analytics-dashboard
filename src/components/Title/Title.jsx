import React from "react";

const Title = (props) => {
  return (
    <div style={{ marginBottom: 15 }}>
      <label style={{ fontSize: 16, fontWeight: "bold", marginBottom: 5 }}>
        {props.title}
      </label>
      <hr
        style={{
          marginTop: 0,
          position: "absolute",
          border: "2px solid #4a5068",
          width: 50,
        }}
      />
    </div>
  );
};

export default Title;
