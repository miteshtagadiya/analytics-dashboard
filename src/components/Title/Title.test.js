import React from "react";
import renderer from "react-test-renderer";
import Title from "./Title";

test("renders correctly when there is no props", () => {
  const tree = renderer.create(<Title />).toJSON();
  expect(tree).toMatchSnapshot();
});

test("renders correctly when there title prop", () => {
  const tree = renderer.create(<Title title="Title" />).toJSON();
  expect(tree).toMatchSnapshot();
});
