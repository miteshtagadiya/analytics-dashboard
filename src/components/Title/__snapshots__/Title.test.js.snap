// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly when there is no props 1`] = `
<div
  style={
    Object {
      "marginBottom": 15,
    }
  }
>
  <label
    style={
      Object {
        "fontSize": 16,
        "fontWeight": "bold",
        "marginBottom": 5,
      }
    }
  />
  <hr
    style={
      Object {
        "border": "2px solid #4a5068",
        "marginTop": 0,
        "position": "absolute",
        "width": 50,
      }
    }
  />
</div>
`;

exports[`renders correctly when there title prop 1`] = `
<div
  style={
    Object {
      "marginBottom": 15,
    }
  }
>
  <label
    style={
      Object {
        "fontSize": 16,
        "fontWeight": "bold",
        "marginBottom": 5,
      }
    }
  >
    Title
  </label>
  <hr
    style={
      Object {
        "border": "2px solid #4a5068",
        "marginTop": 0,
        "position": "absolute",
        "width": 50,
      }
    }
  />
</div>
`;
